conans/model/build_info.py

```diff
@@ -79,7 +79,7 @@ def __getitem__(self, item):
         return self._dependencies[item]
 
     def __repr__(self):
-        return TXTGenerator(self).content
+        return TXTGenerator(self, CppInfo(os.getcwd())).content
```

Comment:
It is used when building packages locally, to save a "conanbuildinfo.txt" as reference, and the second parameter of the constructor of TXTGenerator is actually not used, it should work with ```None``` too. I'd probably try to replace line 203 of conans/client/installer.py with something like:

```python
save(os.path.join(build_folder, BUILD_INFO), TXTGenerator(conan_file.deps_cpp_info, conanfile.cpp_info))
```

No need to do it here, lets do it in another PR, just try to change CppInfo(os.getcwd()) for None and add a FIXME, or leave it as is, no problem.